Zadanie2.rb

```diff
@@ -0,0 +1,105 @@
+class Character
+  attr_accessor :name, :level
+	
+  def initialize(name: , level:)
+    self.name= name.class==String ? name: "NoName"
+    self.level= (level.class==Integer && level>0 && level<100)? level:1 

```

Comment:
wyrażenia `condition ? do_a : do_a ` używamy tylko do bardzo prostych i krótkich konstrukcji. Jak coś jest bardziej skomplikowane to lepiej pozostać przy `if/else`